readme.md

```diff
@@ -8,23 +8,70 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-[**rehype**][rehype] plugin to automatically add `target` and `rel` attributes
-to external links.
+**[rehype][]** plugin to add `rel` (and `target`) to external links.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`unified().use(rehypeExternalLinks[, options])`](#unifieduserehypeexternallinks-options)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This package is a [unified][] ([rehype][]) plugin to add `rel` (and `target`)
+attributes to external links.
+It is particularly useful when displaying user content on your reputable site,
+because users could link to disreputable sources (spam, scams, etc), as search
+engines and other bots will discredit your site for linking to them (or
+legitimize their sites).
+In short: linking to something signals trust, but you can’t trust users.
+This plugin adds certain `rel` attributes to prevent that from happening.
```

Comment:
The way this is framed, is sounds like `rel` and `target` will take away the search engine hit from linking to a disreputable site.
Is that the case? I'm not sure it is.

Reply:
https://en.wikipedia.org/wiki/Nofollow.

This phrasing is written with https://github.com/rehypejs/rehype-external-links/issues/1 in mind though

Reply:
Hmm, might be better to use `rel="UGC"` in that case, but okay.
It's also worth noting that `nofollow` applies only to PageRank, it could still be considered as a part of other signals.

Reply:
First time I heard of `ugc`! It hasn’t been added to [the spec](https://github.com/wooorm/html-link-types/blob/main/index.js) though (or by the [microformats extensions linked to from the spec](https://github.com/wooorm/a-rel/blob/main/index.js)). I wonder how well it’s supported.
My experience here is based on before 2019 I guess, and based on that GitHub only adds `nofollow`. Hence I think it’s good to match that.
(and being nonstandard and first time I’ve seen it used, it feels a bit weird to add `ugc`?)

Reply:
I also think the statement that it only affects “PageRank” is not giving it enough credit, given that that wikipedia page also lists other algorithms affected by it, which a quick google (hah!) seems to corroborate?

Reply:
It is standard in several major search engines
* https://developers.google.com/search/docs/advanced/guidelines/qualify-outbound-links
* https://www.bing.com/webmasters/help/webmasters-guidelines-30fba23a

https://moz.com/blog/nofollow-sponsored-ugc explains how things have changed.
`rel="nofollow ugc"` can be used in combination if preferred as well.

Reply:
That is not how *standard* works. Google (or Microsoft) are not standard. The HTML standard, including the by them appointed registry, is standard.
My point remains that `ugc` is not standard and that there is no precedent (i.e., GitHub).

Regardless, this is a very different conversation than whether `nofollow` does things?

Reply:
I feel like the goal posts of this conversation are changing. It was about wording. Now it’s about adding a new default because it (arguably) might be better. There’s a different issue open on changing the defaults already, so that can be discussed there regardless of these docs?